### PR TITLE
Add rules to check if else and media queries

### PR DESCRIPTION
**Pull request by alexander-schranz**
If else inspired by: https://github.com/kristerkari/stylelint-scss/blob/master/docs/examples/if-else.md

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -112,7 +127,12 @@ module.exports = {
         "scss/at-else-if-parentheses-space-before": "always",
         "scss/at-function-parentheses-space-before": "never",
         "scss/at-function-pattern": "^[a-z][a-z-A-Z0-9]*$",
-        "scss/at-mixin-parentheses-space-before": "never"
+        "scss/at-mixin-parentheses-space-before": "never",
+        "scss/at-else-closing-brace-newline-after": "always-last-in-chain",
+        "scss/at-else-closing-brace-space-after": "always-intermediate",
+        "scss/at-else-empty-line-before": "never",
+        "scss/at-if-closing-brace-newline-after": "always-last-in-chain",
+        "scss/at-if-closing-brace-space-after": "always-intermediate",
```
Comment (c00n84) [inline index.js]: @alexander-schranz remove last comma.

File: test/each.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+$test1: (
+    desktop: 200px,
+    tablet: 350px,
+    mobile: 400px,
```
Comment (c00n84) [inline test/each.scss:5]: @alexander-schranz remove comma in last line.

File: test/each.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+);
+
+$test2: (
+    desktop: 100px,
+    tablet: 150px,
+    mobile: 200px,
```
Comment (c00n84) [inline test/each.scss]: @alexander-schranz remove comma in last line.

**Comment by c00n84** [comment]
@alexander-schranz Check CI fails and review comments - else ACK

File: test/each.scss (hunk 2 above)
Comment (alexander-schranz) [reply test/each.scss:5]: @c00n84 sure? for git it is better and in php we also have this comma

/cc @thomasduenser @dominikmatt @sabinebaer @rs2487

Comment (thomasduenser) [reply test/each.scss:5]: @alexander-schranz @c00n84 Hard decision. It feels strange in SCSS, but I think it's a good idea to handle it the same way as in PHP as it's easier to review.

**Pull request merged**